test/Microsoft.Azure.Devices.Edge.Test.Common/linux/LinuxStandardBase.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Linux
+{
+    using Microsoft.Azure.Devices.Edge.Util;
+    using SupportedPackageExtension = PackageManagement.SupportedPackageExtension;
```

Comment:
Nit: The higher level class (`PackageManagement`) should depend on the lower level class (`LinuxStandardBase`), not the other way around.

Reply:
Ok.